metadata-service/factories/src/main/java/com/linkedin/metadata/boot/factories/IngestRetentionPoliciesStepFactory.java

```diff
@@ -27,13 +27,16 @@ public class IngestRetentionPoliciesStepFactory {
   @Value("${entityService.retention.enabled}")
   private Boolean _enableRetention;
 
+  @Value("${entityService.retention.applyOnBootstrap:false}")
```

Comment:
Can we add this to application.yml as well so we can change using an environment variable?

Reply:
done